http-update.md

```diff
@@ -0,0 +1,24 @@
+# Triggering version updates over HTTP
+
+To queue an update of your package you can use the `POST /api/packages/:packageName/update` endpoint.
+
+## `POST /api/packages/:packageName/update`
+
+Queues an update for the specified package.
+
+Query params:
+`secret`: string (optional) provide the secret as query
+`header`: string (optional) provide which header is used to check the secret (must start with X-)
+
+Body params: (application/x-www-form-urlencoded, multipart/form-data or application/json)
+`secret`: string (optional) provide the secret as body param
```

Comment:
Hmm, what's the use case for this? Isn't this prone to be over-used/falsely used?
(I'm referring to the API method itself)

Reply:
I wanted to make sure it's easy to incoperate this into any service you might be using for webhooks, so I'm offering to make the API callable with a secret per query string, body param or per HTTP Header.